ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -146,11 +146,12 @@ object JdbcIndexer {
       for {
         // Note: the LedgerDao interface is only used for initialization here, it can be released immediately
         // after initialization is finished. Hence the use of ResourceOwner.use().
+        // TODO append-only: Rewrite the initialization using the StorageBackend interface, and remove the use of LedgerDao from this class
         _ <- com.daml.platform.store.appendonlydao.JdbcLedgerDao
           .writeOwner(
             serverRole,
             config.jdbcUrl,
-            config.databaseConnectionPoolSize,
+            connectionPoolSize = 1,
```

Comment:
`LedgerDao` is only used for sequential initialization here, 1 connection at a time is enough